doc/tutorial/score_card.md

```diff
@@ -0,0 +1,114 @@
+# Credit ScoreCard Model on SQLFlow
+
+The credit scorecard model is widely used in financial lending scenario, the model outputs a score that represents
+how likely the lender will be repaid on time. This tutorial introduces how to train a credit scorecard model with SQLFlow.
+
+## Dataset Introduction
+
+This tutorial using a public dataset [Give Me Some Credit](https://www.kaggle.com/c/GiveMeSomeCredit/data)
+on the [Kaggle](https://www.kaggle.com) website. The dataset contains 150,000 rows training data,
+each row includes 11 columns, where `serious_dlqin2yrs` is the target column, represents person experienced 90 days
+past due delinquency or worse. The description for each column as the table:
```

Comment:
`The description for each column as the table` -> `The description for each column is as the table`

Reply:
It seems that the comment is not resolved.